src/lib.rs

```diff
@@ -66,6 +66,25 @@ where
         self.0.status
     }
 
+    pub fn command_strobe(&mut self, command_strobe: CommandStrobe) -> Result<(), Error<SpiE>> {
+        let command = match command_strobe {
+            CommandStrobe::ResetChip => Command::SRES,
+            CommandStrobe::EnableAndCalFreqSynth => Command::SFSTXON,
+            CommandStrobe::TurnOffXosc => Command::SXOFF,
+            CommandStrobe::CalFreqSynthAndTurnOff => Command::SCAL,
+            CommandStrobe::EnableRx => Command::SRX,
+            CommandStrobe::EnableTx => Command::STX,
+            CommandStrobe::ExitRxTx => Command::SIDLE,
+            CommandStrobe::StartWakeOnRadio => Command::SWOR,
+            CommandStrobe::EnterPowerDownMode => Command::SPWD,
+            CommandStrobe::FlushRxFifoBuffer => Command::SFRX,
+            CommandStrobe::FlushTxFifoBuffer => Command::SFTX,
+            CommandStrobe::ResetRtcToEvent1 => Command::SWORRST,
+            CommandStrobe::NoOperation => Command::SNOP,
+        };
+        Ok(self.0.write_cmd_strobe(command)?)
+    }
```

Comment:
Think it would make for a better API if these were distinct functions. It would compile to the same machine code in the end, and maybe there's more behavior to encode around them. The low level API exposes all the above information already.

Reply:
Do you mean that for each enum field we should have a dedicated function invoked by the user, instead of `command_strobe(...)`?
```
command_strobe(CommandStrobe::ResetChip) -> reset_chip()
command_strobe(CommandStrobe::EnableRx) -> enable_rx()
command_strobe(CommandStrobe::EnableTx) -> enable_tx()
and so on ...
```
If so, I had this idea in mind as well, but after some trial, decided to go with the enum (perhaps due to code compactness).
The main idea here is to offer user a human readable API to act on.
We can still try with the functions or something even more exotic:
```
self.command_strobe(CommandStrobe::ResetChip) -> self.command.reset_chip()
self.command_strobe(CommandStrobe::EnableRx) -> self.command.enable_rx()
self.command_strobe(CommandStrobe::EnableTx) -> self.command.enable_tx()
...
```
😄

Reply:
While it is a bit more verbose, it's hopefully just implemented once, and I think it makes for a prettier API that keeps a more high level feel to it, and once the optimizer has run, the end result will be the same.